libs/shared/ui-elements/src/lib/prio-button/prio-button.component.ts

```diff
@@ -9,10 +11,20 @@ import { Component, OnInit, Input, Output, EventEmitter, HostBinding } from '@an
 
 export class PrioButtonComponent implements OnInit {
 
-  inputFocus = false;
+  @Input() style = 'solid' ;  //solid || outline || flat
```

Comment:
We should rely on TS Types so we should define the available styles like this 
`export type ButtonType = "solid" | "outline" | "flat"; `
`@Input() style: ButtonType  = 'solid' `

Reply:
done